src/components/Badge/Badge.tsx

```diff
@@ -3,57 +3,67 @@ import styles from './Badge.scss';
 import {Typography} from '~/components/Typography';
 import classnames from 'clsx';
 
+type TBadgeColor = 'accent' | 'success' | 'danger';
+
 export enum BadgeColor {
-    'accent' = 'accent',
-    'success' = 'success',
-    'danger' = 'danger'
+    ACCENT = 'accent',
+    SUCCESS = 'success',
+    DANGER = 'danger'
 }
 
+type TBadgeType = 'round' | 'diamond';
+
 export enum BadgeType {
-    'round' = 'round',
-    'diamond' = 'diamond'
+    ROUND = 'round',
+    DIAMOND = 'diamond'
 }
 
-type BadgeProps = {
+interface IBadgeProps {
     /**
      * Badge label, only for type round
      */
-    label?: string,
+    label?: string;
 
     /**
      * Badge color
      */
-    color?: BadgeColor,
+    color?: TBadgeColor;
```

Comment:
Why can't we use directly `color?: BadgeColor` and `type?: BadgeType` ?

Reply:
I noticed when using enums for the type, the editor suggestions for the type don't show the values available (e.g., accent or success or danger), it would just show the type "BadgeColor" which isn't super useful.

Once I changed it to a type, the values did appear in the editor popup.

Reply:
<img width="396" alt="Screen Shot 2020-08-20 at 10 50 18 AM" src="https://user-images.githubusercontent.com/1192335/90791494-431c5e00-e2d7-11ea-8c5c-9cca2b19caf8.png">
<img width="388" alt="Screen Shot 2020-08-20 at 11 04 29 AM" src="https://user-images.githubusercontent.com/1192335/90791507-46174e80-e2d7-11ea-953f-57fe131d854c.png">